src/main/features/core/websocketAPI.js

```diff
@@ -76,7 +76,10 @@ const enableAPI = () => {
     });
     Emitter.sendToWindowsOfName('main', 'openport:request');
   }
-  server = new WebSocketServer({ port: global.API_PORT || process['env'].GPMDP_API_PORT || 5672 }, () => { // eslint-disable-line
+  server = new WebSocketServer({
+    host: global.API_HOST || process.env.GPMDP_API_HOST || '0.0.0.0',
+    port: global.API_PORT || process.env.GPMDP_API_PORT || 5672,
```

Comment:
This NEEDS to be

```
process['env'].GPMDP_API_HOST
process['env'].GPMDP_API_PORT
```

You've got to be careful when changing code in a project where you don't understand the build tooling.  We inline replace `process.env.XXXXX` with the **build machines** environment variables.  If we want to use the client's environment variables we have to use this syntax.

See https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/master/gulpfile.babel.js#L156